AEM/tf-itex-cheat-sheet.md

```diff
@@ -1,244 +1,73 @@
-
-export KMP_SETTINGS=TRUE
-
-
-
-Intel Extension for TensorFlow
-This extension provides the most up-to-date features and optimizations on Intel hardware, most of which will eventually be upstreamed to stock TensorFlow releases. Additionally, while users can get many optimization benefits by default without needing an additional set up, Intel Extension for TensorFlow provides further tuning and custom operations to boost performance even more.
-
-For additional installation methods, see the Intel® Extension for TensorFlow* Installation Guide.
-
-For more information, see Intel Extension for TensorFlow*.
-
-Basic GPU Installation using PyPI
-
-pip install --upgrade intel-extension-for-tensorflow[gpu]
-
-Import Intel Extension for TensorFlow
-
-import intel_extension_for_tensorflow as itex
-
-Get an XPU Back End Type
-
-itex.get_backend()
-
-Toggle a GPU Back End (in the Code): Set by Default
-
-itex.set_backend(‘GPU’)
-
-Toggle a GPU Back End (Command Prompt): Set by Default
-
-Export ITEX_XPU_BACKEND="GPU"
-
-Advanced Automatic Mixed Precision (in the Code): A Basic Configuration with Improved Inference Speed with Reduced Memory Consumption
-
-auto_mixed_precision_options = itex.AutoMixedPrecisionOptions()
-
-
-
-auto_mixed_precision_options.data_type = itex.BFLOAT16 # or itex.FLOAT16
-
-Advanced Automatic Mixed Precision (Command Prompt): A Basic Configuration with Improved Inference Speed with Reduced Memory Consumption
-
-export ITEX_AUTO_MIXED_PRECISION=1 
-
-export ITEX_AUTO_MIXED_PRECISION_DATA_TYPE="BFLOAT16" # or "FLOAT16"
-
-Customized AdamW Optimizer (in the Code)
-
-itex.ops.AdamWithWeightDecayOptimizer(
-
-    weight_decay_rate=0.001,
-
-    learning_rate=0.001, beta_1=0.9,
-
-    beta_2=0.999,
-
-    epsilon=1e-07, name='Adam',
-
-    exclude_from_weight_decay=["LayerNorm",
-
-    "layer_norm", "bias"], **kwargs
-
-)
-
-
-
-Customized Layer Normalization (in the Code)
-
-itex.ops.LayerNormalization(
-
-    axis=-1, epsilon=0.001, center=True,
-
-    scale=True,
-
-    beta_initializer='zeros',
-
-    gamma_initializer='ones',
-
-    beta_regularizer=None,
-
-    gamma_regularizer=None,
-
-    beta_constraint=None,
-
-    gamma_constraint=None, **kwargs
-
-)
-
-Customized GELU (in the Code)
-
-itex.ops.gelu(
-
-    features, approximate=False, name=None
-
-)
-
-Customized LSTM (in the Code)
-
-itex.ops.ItexLSTM(
-
-    200, activation='tanh',
-
-    recurrent_activation='sigmoid',
-
-    use_bias=True,
-
-    kernel_initializer='glorot_uniform',
-
-    recurrent_initializer='orthogonal',
-
-    bias_initializer='zeros', **kwargs
-
-)
-
-
-
+<br><br>
+## Intel® Optimization for TensorFlow*: A Public Release from Google
+Features and optimizations for TensorFlow* on Intel hardware are frequently upstreamed and included in stock TensorFlow* releases. As of TensorFlow* v2.9, Intel® oneAPI Deep Neural Network Library (oneDNN) optimization is automatically enabled.
+
+For more information, see [TensorFlow](https://www.tensorflow.org/).
```

Comment:
possible to add this article https://www.intel.com/content/www/us/en/developer/articles/guide/getting-started-with-automixedprecisionmkl.html as a reference?

Reply:
Adding the link